asmcomp/cmmgen.ml

```diff
@@ -3082,6 +3082,31 @@ let global_data name v =
   Cdata(emit_structured_constant (name, Global)
           (Uconst_string (Marshal.to_string v [])) [])
 
+let module_map_debug = try
+                         ignore (Sys.getenv "CAML_MODMAP_DEBUG"); true
+  with Not_found -> false
```

Comment:
Yeah, I should have written WIP on this PR. This is still debugging code that should be removed before the merge.